pin telescope version (avoids broken project.nvim)

lytex opened the pull request:
`:Telescope projects` is not doing anything so that `project.nvim` has stopped working

ChristianChiarulli reviewed (commented):
We can make the mappings work like this:

```
<cmd>lua require('telescope').extensions.projects.projects()<cr>
```

lytex committed:
Fix find project keymapping

Call :lua require('telescope').extensions.projects.projects() instead of
:Telescope projects
